js/id/renderer/features.js

```diff
@@ -100,10 +100,34 @@ iD.Features = function(context) {
         );
     }, 250);
 
+    defineFeature('indoor', function isIndoorOther(entity) {
+        return !!entity.tags.level;
+    });
+
+    defineFeature('indoor_different_level', function isHiddenByLevel(entity, resolver, geometry) { //disabled in indoor_mode -> hides unwanted levels
+        var current = context.indoor().level();
+
+        if (entity.tags.level && !inRange(current, entity.tags.level) && !inRange(current, entity.tags.repeat_on))
+            return true;
+
+        if (geometry === 'point' && !entity.tags.level)
+            return true;
+
+        if (entity.tags.max_level && entity.tags.min_level) {
+            if (parseFloat(entity.tags.max_level) < current || parseFloat(entity.tags.min_level) > current)
+                return true;
+        }
+        else if (entity.tags.building || entity.tags['building:part']) {
+            if (current < 0 || current >= parseFloat(entity.tags['building:levels'] || 0))  //one level <=> level=0
+                return true;
+        }
+    });
+
```

Comment:
Now it shows all the time in the "map features" panel. But maybe it doesnt make sense to show this particular feature. It could be added/removed when indoor is toggled, but i like it this way better. (Also something is using it lower down in this file..)

Conclusion - maybe just hide the checkbox when indoor mode is not active?